src/plugins/intel_cpu/src/nodes_factory.cpp

```diff
@@ -2,108 +2,107 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "nodes/reference.h"
-#include "nodes/shapeof.h"
-#include "nodes/batch_to_space.h"
-#include "nodes/multiclass_nms.hpp"
 #include "nodes/adaptive_pooling.h"
+#include "nodes/batch_to_space.h"
```

Comment:
Just an ask for the future: please avoid changes caused by your local dev environment settings.

Reply:
Noted